docs/tutorials/tutorial-query.md

```diff
@@ -57,12 +58,17 @@ You can issue the above query from the console.
 
 ![Query autocomplete](../assets/tutorial-query-01.png "Query autocomplete")
 
-The console query view provides autocomplete together with inline function documentation.
-You can also configure extra context flags to be sent with the query from the more options menu.
+The console query view provides autocomplete functionality with inline documentation.
 
 ![Query options](../assets/tutorial-query-02.png "Query options")
 
-Note that the console will by default wrap your SQL queries in a limit so that you can issue queries like `SELECT * FROM wikipedia` without much hesitation - you can turn off this behaviour.
+You can also configure extra context flags to be sent with the query from the more options menu.
+
+Note that the console will by default wrap your SQL queries in a limit where appropriate allowing you to issue queries like `SELECT * FROM wikipedia` without hesitation - you can turn off this behaviour from the `Smart query limit` toggle.
```

Comment:
Note that the console will (by default) wrap your SQL queries in a limit where appropriate so that queries such as `SELECT * FROM wikipedia` can complete. You can turn off this behaviour from the `Smart query limit` toggle.